generic/tests/cfg/unattended_install.cfg

```diff
@@ -122,13 +122,23 @@
             no WinXP Win2000 Win2003 WinVista
             unattended_delivery_method = cdrom
             cdroms += " unattended"
-            Windows:
-                i440fx:
-                    cd_format_unattended = ide
-                q35:
-                    cd_format_unattended = ahci
             drive_index_unattended = 1
             drive_index_cd1 = 2
+            variants:
+                - @default:
+                    Windows:
+                        i440fx:
+                            cd_format_unattended = ide
+                        q35:
+                            cd_format_unattended = ahci
+                # Support "scsi_cd" drive format for cdrom_cd
+                # and cdrom_winutils in Windows
+                - with_scsi_cd:
+                    only Windows
```

Comment:
Should support for linux too.
For linux installation, please set cdrom_format=scsi-cd for all cdroms.

Thanks.